.circleci/config.yml

```diff
@@ -512,7 +512,7 @@ jobs:
   spm-revenuecat-ui-ios-16:
     executor:
       name: macos-executor
-      xcode_version: '14.3.0'
+      xcode_version: '15.4.0'
 
```

Comment:
this one didn't _need_ Xcode 14, and we do intend to move to 15, so I figured it was worth moving it over